.github/workflows/build.yml

```diff
@@ -18,17 +18,19 @@ jobs:
         with:
           python-version: 3.9
 
+      - name: Install poetry
+        uses: abatilo/actions-poetry@v2
```

Comment:
what do you think about `snok/install-poetry`? Looks like it has more recent updates and more stars, it's minor but I would imagine that it would be better to use for long-term

Reply:
I noticed that in `upload.yml` you replaced python installation step, so I suppose here it also could be deleted

Reply:
switched to `snok/install-poetry`. actually was a mistake to remove the python install. added back